Sparkle/SUHost.m

```diff
@@ -120,7 +120,9 @@ - (NSString *__nonnull)name
 
 - (NSString *__nonnull)version
 {
-    NSString *version = [self.bundle objectForInfoDictionaryKey:(__bridge NSString *)kCFBundleVersionKey];
+    NSString *version = [self.bundle objectForInfoDictionaryKey:@"CFBundleShortVersionString"];  // bt_browser added.  prefer long version string
+    if (!version || [version isEqualToString:@""])  // bt_browser fall back to CFBundleVersion
```

Comment:
fall back cause it is what was there in the first place.  want me to take out the fallback?